packages/browser/test/e2e/scenarios/initializations/default.js

```diff
@@ -0,0 +1,29 @@
+import * as Sentry from '@sentry/browser';
+import { normalize } from '@sentry/utils';
+
+window.Sentry = Sentry;
+window.events = [];
+
+Sentry.init({
```

Comment:
ok I know this probably adds a ton of boilerplate, so maybe the next step here is to split this PR up so that we only merge the playwright stuff with a couple of tests + then add the rest of the tests afterwards in other PRs.

Can we change the folder structure so that it's something like this:

```
scenario
|___ index.hbs -> contains scenario so the html + script lives right next to each other. Also contains SDK init
|___ test.js -> contains assertions
|___ fixture1.json -> optional fixtures (
|___ fixture2.json
```

By having the html, the sdk init, and scenario be right next each other, it becomes a lot easier to understand what is going on (and what specifically is being tested).

What do you think?

Reply:
Yes, I also think the current structure is a bit hard to follow when reading tests.

But I have a few concerns about this structure:
 - Almost every test case has a separate `subject / script`, but most of the initialization logic and templates are common between cases. (It's the case with the current `browser` tests, but I suppose a similar situation might be on `react` / `vue` / `angular` etc. tests as well, maybe not on `integrations` or `core` though.) Switching to this structure might initiate lots of duplication with minimal changes, making it easier to read, but harder to write. We may get over with a fallback logic (like if a specific template / initialization is not defined for a test, it may fall back to the default template defined in the parent folder).
 - Merging template / init / subject into one HTML for each test might be a bit error-prone when writing new tests as a minimal defect in structure / order of the HTML template (like adding / forgetting to put `defer` in a specific script) can create flakiness and can be time-consuming to debug and fix. So I would rather keep those separately defined and have base defaults, but structured in a more intuitive way. What do you think?

So, I'm suggesting something similar, but keeping definitions separate like:

```
tests/
|---- temp_default.hbs [global fallback]
|---- init_default.hbs [global fallback]
|---- breadcrumbs/
      |---- temp_default.hbs [optional fallback for breadcrumb tests]
      |---- init_default.js [optional fallback for breadcrumb tests]
      |---- click_event_tree/
            |---- template.hbs [optional case specific template]
            |---- init.js [optional case specific init]
            |---- subject.js [required]
            |---- test.ts [assertions]
            |---- fixtures_0.json [optional]
```

 How does it sound?

Reply:
Yup I like that setup, let's do that 👍 

I think we can even forgo the global fallback and just have each category have a default (that way the inheritance tree will only ever be 1 level deep).

Reply:
@AbhiPrasad, switched to this setup with 3 tests. If it looks good, I'll port the remaining tests.

Reply:
@onurtemizkan can we break up the PR so that we only need to merge in playwright initial stuff + those 3 tests? We can get to the rest of the tests afterwards. Makes it easier to review.

Reply:
Also, mind writing a README on how it all works?